Odata-docs/odatalib/v7/edm/iedmtargetpath-for-annotations-target.md

```diff
@@ -0,0 +1,463 @@
+---
+title: "Use IEdmTargetPath for out-of-line annotations target"
+description: "Use IEdmTargetPath for out-of-line annotations target"
+author: kenitoinc
+ms.author: kemunga
+ms.date: 05/2/2023
+ms.topic: article
+ 
+---
+# Use IEdmTargetPath for out-of-line annotations target
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../../includes/appliesto-odatalib-v7.md)]
+
+The Edm library lets you [define annotations](/odata/odatalib/edm/define-annotations).
+
+[This](/odata/odatalib/edm/set-annotations-schema) documentation describes how to define **out-of-line** annotations that targets a model element.
+
+## Background
+
+Below is a simple Edm model with an out-of-line annotation that targets a model element.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+    <edmx:DataServices>
+        <!-- Model elements -->
+        <Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+            <EntityContainer Name="Container">
+                <EntitySet Name="Employees" EntityType="Ns.Employee" />
+            </EntityContainer>
+            <Annotations Target="Ns.Employee">
+                <Annotation Term="Ns.MyTerm" String="Name OutOfLine MyTerm Value" />
+            </Annotations>
+        </Schema>
+    </edmx:DataServices>
+</edmx:Edmx>
+```
+
+`IEdmTargetPath` allows us to define out-of-line annotations that target a path to a model element.
+
+Below are examples of paths to a model element.
```

Comment:
You want to introduce what the feature you implemented will help someone achieve:
```suggestion
The path expression must resolve to a model element in the entity model which means it can comprise of multiple segments. The path expression may be:
- Qualified name of a schema child - e.g., `Ns.Employee` from the metadata above
- Qualified name of a schema child following by a forward slash and name of child element
- Qualified name of a structured type followed by zero or more property, navigation property, or type cast segments, each segment starting with a forward slash - e.g., `Ns.Employee/Name` from the sample metadata above.
- Qualified name of an entity container followed by a segment containing a singleton or entity set name and zero or more property, navigation property, or type segments - e.g., `Default.Container/Employees/Name` from the sample metadata above. 

Below are examples of path expressions to model elements.
```

Reply:
Paths to a model always begin from the Entity container

https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_TargetPath